lib/utils/isStandardSyntaxKeyframesName.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const hasInterpolation = require('./hasInterpolation');
+
+/**
+ * Check whether a keyframes name is standard
+ *
+ * @param {string} keyframesName
+ * @returns {boolean}
+ */
+
+module.exports = function (keyframesName) {
```

Comment:
[suggest] Can you please give a function name for debuggability (avoiding anonymous functions)?

```suggestion
module.exports = function isStandardSyntaxKeyframesName(keyframesName) {
```